src/test/java/com/werfen/report/CompareExcelFiles.java

```diff
@@ -0,0 +1,105 @@
+package com.werfen.report;
+
+
+import org.apache.poi.ss.usermodel.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class CompareExcelFiles {
+
+    public void verifyIfExcelFilesHaveSameNumberAndNameOfSheets(Workbook workbook1, Workbook workbook2) {
```

Comment:
Just as general rule for all methods in this class, I would use the nomenclature "actual" and "expected" instead of "item1" and "item2" which allows creating more understandable assertion messages